tests/test_schema.py

```diff
@@ -1780,6 +1780,22 @@ class Model(BaseModel):
     }
 
 
+def test_literal_enum():
+    class MyEnum(str, Enum):
+        FOO = 'foo'
+        BAR = 'bar'
+
+    class Model(BaseModel):
+        kind: Literal[MyEnum.FOO]
+
+    assert Model.schema() == {
+        'title': 'Model',
+        'type': 'object',
+        'properties': {'kind': {'title': 'Kind', 'enum': ['foo'], 'type': 'string'}},
+        'required': ['kind'],
+    }
+
```

Comment:
Should we also test `Model.json_schema()`?

Reply:
IMO no need. It's just the json dump of the dict. Most cases test only `schema()` for this reason